packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -174,11 +180,15 @@ function OrgSelect({
     setValue(val);
 
     if (onChangeProp) {
-      onChangeProp(value, e);
-      return;
+      return onChangeProp(value, e);
```

Comment:
Hmm... This looks like we may not want to return anything since it's a change handler. Any reason we shouldn't just remove the return statement entirely, especially since we only return conditionally here AFAICT?

Reply:
You can return false from change handlers to effectively do e.preventDefault and e.stopPropagation.

Reply:
🤦‍♂ You're right – I completely forgot that's a thing you can do